code/modules/mob/living/living.dm

```diff
@@ -279,6 +279,8 @@
 		return FALSE
 	if(SEND_SIGNAL(src, COMSIG_LIVING_TRY_PULL, AM, force) & COMSIG_LIVING_CANCEL_PULL)
 		return FALSE
+	if(SEND_SIGNAL(AM, COMSIG_LIVING_TRYING_TO_PULL, src, force) & COMSIG_LIVING_CANCEL_PULL)
```

Comment:
isnt this what can_be_pulled does?

Reply:
No, `can_be_pulled` handles mobility/move resistance checks to see if the atom is valid to be moved. 

It's akin to `can_inject` vs `try_inject`. Shouldn't mix the two.